basxconnect/mailer_integration/views.py

```diff
@@ -0,0 +1,34 @@
+import bread.layout.components.notification
+import htmlgenerator as hg
+from bread.layout.components.form import Form
+from bread.utils import aslayout
+from django import forms
+
+from basxconnect.mailer_integration import download_data
+from basxconnect.mailer_integration.mailchimp import datasource
+
+
+@aslayout
+def mailchimp_view(request):
+    if request.method == "POST":
+        try:
+            sync_result = download_data.download_persons(
+                datasource.MailchimpDatasource()
+            )
+            notification = bread.layout.components.notification.InlineNotification(
+                "Success",
+                f"Synchronized mailing preferences for {sync_result.total_synchronized_persons} Mailchimp "
+                f"contacts. {sync_result.new_persons} new persons were added to the BasxConnect database.",
+            )
+        except Exception as e:
+            notification = bread.layout.components.notification.InlineNotification(
+                "Error", "An error occured during synchronization.", kind="error"
+            )
```

Comment:
Could we add the exception trace to the notification? Even when it does not directly help the user, it is very usefull during debugging and when user report a problem.

Reply:
Otherwise the error goes completely silent except the notification message.